fullstack/js/price-calculator/app/managers/base-price-manager.js

```diff
@@ -0,0 +1,25 @@
+import ProductBasePrice from '../models/base-price'
+
+export default class BasePriceManager {
+  data = {}
+
+  constructor(jsonFile) {
+    this.data = this.readJson(jsonFile)
+  }
+
+  readJson(jsonFile) {
+    let data = {}
+    const rawData = require(jsonFile)
+
+    for (const rawItem in rawData) {
```

Comment:
Should this be `of`?

Reply:
According to MDN `of` can iterate Array, Object, Map, Set, arguments while `in` only arrays.
Since `rawData` is array we can leave this as it is ?
Source: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Loops_and_iteration#for...of_statement

Reply:
Somehow I thought that `rawData` is an object. Good to me then